test/ext/format-text.js

```diff
@@ -0,0 +1,33 @@
+// From https://github.com/azer/format-text
+
+module.exports = format;
+
+function format(text) {
+  var context;
+
+  if (typeof arguments[1] == 'object' && arguments[1]) {
+    context = arguments[1];
+  } else {
+    context = Array.prototype.slice.call(arguments, 1);
+  }
+
+  return String(text).replace(/\{?\{([^{}]+)}}?/g, replace(context));
+};
+
+function replace (context, nil){
+  return function (tag, name) {
+    if (tag.substring(0, 2) == '{{' && tag.substring(tag.length - 2) == '}}') {
```

Comment:
I like to put each condition on its own line for clarity e.g.:

```
if (
  tag.substring(0, 2) == '{{' && 
  tag.substring(tag.length - 2) == '}}'
) {
```

Also, triple equals.